### PR TITLE
feat(homepageDesign): #82 Designed the homepage of tribe

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
-  Designed the homepage of the tribe.
Screenshot from 2019-05-19 00-43-59

### CODE REVIEW

File: web/src/app/components/social-cards/social-cards.component.ts [hunk 1]
```diff
@@ -81,7 +80,7 @@ export class SocialCardsComponent {
       });
     } else {
       dialogRef = this.matDialog.open(AlertComponent, {
-        data: { message: 'This is primary account.Do you want to remove it ?' }
+        data: { message: 'This is your primary account.Do you want to remove it ?' }
```
Comment (eddiejaoude) [inline web/src/app/components/social-cards/social-cards.component.ts]: Space needed after `.`, from `account.Do` to `account. Do`

File: web/src/styles/variables.scss [hunk 2]
```diff
@@ -6,6 +6,9 @@ $instagram-primary: #f8f8f84d;
 $text-color-primary: #000000;
 $text-color-secondary: #010311;
 $text-color-accent: #ffffff;
+$primary-accent: #FF5A00;
+$secondary-accent: #ff5a0008;
```
Comment (eddiejaoude) [inline web/src/styles/variables.scss:10]: This doesn't look like a valid colour?

Comment (VenkatVenkatesh) [reply web/src/styles/variables.scss:10]: @eddiejaoude Color code with opacity will be in this format.

Comment (eddiejaoude) [reply web/src/styles/variables.scss:10]: Oh ok, thanks for clarifying 👍

**Comment by VenkatVenkatesh** [comment]
@eddiejaoude Please review and merge.

**Comment by VenkatVenkatesh** [comment]
@eddiejaoude Gentle reminder for merging the branch.

**Comment by eddiejaoude** [comment]
> @eddiejaoude Gentle reminder for merging the branch.

Thank you 👍 🤓

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**